pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
I think you have a typo here, and it should be
```suggestion
    return x / y
```